docs/en-US/component/transfer.md

```diff
@@ -43,43 +43,78 @@ transfer/prop-alias
-| props                 | prop aliases for data source                                                                                                                                                                                                                                                       | `{ key, label, disabled }`        | —                         | —                                                                         |
-| left-default-checked  | key array of initially checked data items of the left list                                                                                                                                                                                                                         | array                             | —                         | [ ]                                                                       |
-| right-default-checked | key array of initially checked data items of the right list                                                                                                                                                                                                                        | array                             | —                         | [ ]                                                                       |
-| validate-event        | whether to trigger form validation                                                                                                                                                                                                                                                 | boolean                           | -                         | true                                                                      |
-
-## Slots
+## API
+
+### Attributes
+
+| Name                  | Description                                                                                                                                                                                                                                                                        | Type                                                            | Default  |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | -------- |
+| model-value / v-model | binding value                                                                                                                                                                                                                                                                      | ^[object]`TransferKey[]`                                        | []       |
```

Comment:
Can the transfer type be linked to the type declaration below, otherwise others may not be able to understand it. 🤔

Reply:
I have defined `TransferKey` in Type Declarations, line 97 of `transfer.md`. Do you mean this?

Reply:
Yes, what I mean is to add a link, click `TransferKey` to jump to the following type declaration.

Reply:
Sorry I don't know how to implement this yet, Can you give me a hint ? Now I can only add another link to the declaration. Like this:
```md
^[object]`TransferKey[]` [Type Declarations](#type-declarations)
```

Reply:
Is there any other way to add link besides the one mentioned by @wzc520pyfm ?  🤔
cc  @tolking @chenxch

Reply:
If written like this, it seems cumbersome for users to view the type.

`object ` -> `TransferKey[]` -> `(string | number)[]`

Perhaps directly specifying the type would make it easier for users to view.

```
^[array]`Array<string | number>`
```

Reply:
Maybe @tolking is right, `Transferkey` is just a simple union type.

Reply:
Agree with @tolking .